subsys/bluetooth/controller/hci_driver.c

```diff
@@ -332,6 +332,7 @@ static void event_packet_process(uint8_t *hci_buf)
 		}
 
 		BT_ERR("No event buffer available");
+		k_oops();
```

Comment:
That's a bad idea; in general you want to keep the application running instead of crashing it when one subsystem has encountered an error. Perhaps you could add an assert instead, which at least the user has some control over.

Reply:
Yeah. I've realized anyway that the zephyr API documentation I was looking at is wrong, so I'll drop this